src/panels/config/integrations/integration-panels/zha/zha-config-dashboard.ts

```diff
@@ -46,6 +46,8 @@ import {
 } from "../../../../../data/zha";
 import { showAlertDialog } from "../../../../../dialogs/generic/show-dialog-box";
 
+const MULTIPROTOCOL_ADDON_URL = "socket://core-silabs-multiprotocol:9999";
```

Comment:
This will not catch 100% of the cases right? It can be an IP, other hostname, other port whatever?